src/components/Accordion/Accordion.js

```diff
@@ -32,7 +40,7 @@ const ModdedIcon = styled(Icon)`
   &:after {
     padding-top: 0;
     padding-bottom: 0;
-    font-size: ${({ theme }) => theme.accordion.iconFontSize};
+    font-size: ${({ mods }) => mods.small ? '14px' : '1em'};
```

Comment:
Same comment as Label, pulling `14px` and `1em` from a theme value would be preferrable